SOURCES/etc/xapi.d/plugins/2crsi-sensors.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import json
+import sys
+import XenAPIPlugin
+import re
+import subprocess
+
+sys.path.append('.')
+from xcpngutils import configure_logging, run_command, error_wrapped
+from xcpngutils.operationlocker import OperationLocker
+
+@error_wrapped
+def get_data(session, args):
+    data = subprocess.check_output(["ipmi-sensors", "-Q", "--ignore-not-available-sensors", "--ignore-unrecognized-events"])
```

Comment:
Can't we use `ipmitool sensor` to get the same info? The thing is that freeipmi is not installed by default while ipmitool is. I tested on one machine and it looks like we have even more information with `ipmitool sensor`. Can you confirm that you also have all infos with your HW?